README.md

```diff
@@ -31,8 +31,16 @@ DNF5 is written in C++ and it can interface with several programming languages:
  * C - not implemented, doesn't seem to be a priority for any of our existing API users
 
 
-:warning: **The current (main) branch is subject of a major rewrite. The API/ABI is currently unstable** :warning:
+How to try DNF5
+---------------
+
 
+| Fedora | Command |
+| --------------------------- | ------- |
+| F36 | `sudo dnf copr enable rpmsoftwaremanagement/dnf5-unstable && sudo dnf install dnf5`[^copr] |
```

Comment:
Minor: maybe I would drop the `F` in the Fedora column values and have just `36`, `37` and `38`?